src/pages/myRegisterdShow/components/registeredcard/RegisteredCard.tsx

```diff
@@ -24,40 +25,69 @@ const RegisteredCard = ({
   };
 
   const getShowTypeText = (key: SHOW_TYPE_KEY): ShowTypes => SHOW_TYPE[key];
+  const calculateDueDate = (dateString: string): number => {
```

Comment:
p5) 가독성을 위해서 띄어쓰기 해주세용
```suggestion

  const calculateDueDate = (dateString: string): number => {
```

Reply:
p5) 그리고 이 함수는 전체 유틸함수로 빼도 될 것 같아요!